src/renderer/components/dex/OrderForm.vue

```diff
@@ -234,7 +236,7 @@ export default {
       }
     },
     estimate() {
-      const holding = this.$store.state.currentMarket ?
+      const holding = this.$store.state.currentMarket && this.$store.state.holdings.length ?
         this.$services.neo.getHolding(this.$store.state.currentMarket.baseAssetId).unitValue :
```

Comment:
Seems like this should just check if the call to getHolding returns null instead of trying to check if the holdings length is not 0.

Reply:
I believe holdings is defaulted to an empty array on init of the store. A null check wouldn’t work in this case.

Reply:
What we are trying figure out of the default value has been changed and we can know if holdings are already populated.